doc/source/intro.rst

```diff
@@ -272,3 +272,19 @@ use_types
   keyword-argument, and attribute type definitions will be skipped.
   This could be turned False for Python3, where Sphinx recognizes
   annotations.
+
+numpy
+''''''
+
+Numpy docstring output configuration options.
+
+use_keyword_section
+  Use **Keyword Arguments** section for all keyword-arguments. Defaults to
+  False. If set to False, all keyword-arguments will be placed under
+  **Parameters** section.
+
+use_types
+  Use types in argument output. Defaults to True. If False, argument,
+  keyword-argument, and attribute type definitions will be skipped.
```

Comment:
Trivial: And return type definitions as well?

Reply:
Hmmm... actually I don't think we skip return types at the moment. I could add that in pretty easily though I think.

Reply:
I'll make an issue for it